code/obj/critter/bee.dm

```diff
@@ -1,3 +1,39 @@
+/obj/critter/domestic_bee/proc/do_hug(mob/user as mob)
+	user.visible_message("<span class='notice'>[src] hugs [user] back!</span>", "<span class='notice'>[src] hugs you back!</span>")
+	if (user.reagents)
+		user.reagents.add_reagent("hugs", 10)
+	return
+
+/obj/critter/domestic_bee/lsbee/do_hug(mob/user as mob)
```

Comment:
please put these procs next to their respective parents

Reply:
all the ones you added at the top that is

Reply:
Hm, having problems doing so. Spawns a bunch of errors in a bunch of unedited files.

Reply:
Ye, shit wack.